wled00/FX_fcn.cpp

```diff
@@ -1012,18 +1030,33 @@ uint32_t WS2812FX::color_from_palette(uint16_t i, bool mapping, bool wrap, uint8
 
 
 //load custom mapping table from JSON file
-void WS2812FX::deserializeMap(void) {
-  if (!WLED_FS.exists("/ledmap.json")) return;
-  DynamicJsonDocument doc(JSON_BUFFER_SIZE);  // full sized buffer for larger maps
+void WS2812FX::deserializeMap(uint8_t n) {
+  String fileName = String(F("/ledmap"));
```

Comment:
I will change this to use `sprintf_P`, which should be a little more heap friendly than String.